microsoft-edge/origin-trials/index.md

```diff
@@ -1,26 +1,172 @@
+
+
+<!-- ====================================================================== -->
+## Submit feedback about an origin trial
+
+
+<!-- ------------------------------ -->
+#### Submit private feedback about an origin trial
+
+To submit private feedback about an origin trial to the browser feature team that owns the origin trial:
+
+1. Go to [Microsoft Edge Origin Trials](https://developer.microsoft.com/microsoft-edge/origin-trials).
+
+1. Select an active origin trial.
+
+1. Click the **Feedback** button.
+
+
+<!-- ------------------------------ -->
+#### Submit public feedback about an origin trial
+
+The GitHub Issues UI allows public discourse about ergonomics and stability of a feature.
+
+To provide public feedback for an origin trial:
+
+1. Go to [Microsoft Edge Origin Trials](https://developer.microsoft.com/microsoft-edge/origin-trials).
+
+1. Select an active origin trial.
+
+1. Click the **Issues** button next to the registration button for the origin trial.
+
+
+<!-- ====================================================================== -->
+## Renew an origin trial token
+
+Origin trial tokens expire every 6 weeks by default.  After the origin trial token expires, you must renew the token.  You can renew the token before it expires.
+
+To renew an origin trial token that has expired or is about to expire:
+
+1. Go to **My Registered Trials** and select an origin trial.
+
+1. Click **Renew**.  A fresh token is sent to you.
+
+1. Copy the new token and use it on your site; see [How to use an origin trial token](#how-to-use-an-origin-trial-token), above.
+
+
+<!-- ====================================================================== -->
+## Developer guidelines
+
+
+<!-- ------------------------------ -->
+#### Allowed origins
+
+Edge origin trials only support SSL-enabled domains. For localhost testing of a feature, please use the appropriate Edge Feature Flag located in `edge://flags`. In the future, we may publish public tokens that are valid for localhost testing. 
+
+
+<!-- ------------------------------ -->
+#### Subdomains, URL paths, and query parameters
+
+
+<!-- ---------- -->
+###### Subdomains
+
+Origin trial tokens can be configured to support any of the following:
+* A single domain, without supporting its subdomains.  For example, register `https://example.com`, and in the **Subdomain Support** section, select the **No** option button.
+* A single subdomain.  For example, register `https://beta.example.com`, and in the **Subdomain Support** section, select the **No** option button.
+* A domain and its subdomains.  For example, register `https://example.com`, and in the **Subdomain Support** section, select the **Yes** option button.
+<!-- todo: check section heading & option button labels at microsoft.com portal; the above ui is from github.io portal -->
```

Comment:
is the section heading at new portal "Subdomain Support"?
is the option button label "Yes" and "No"?
@leahmsft

Reply:
entered comment in PR 3288